index.html

```diff
@@ -184,6 +184,21 @@ <h2> About <br> Myself </h2>
       </div>
     </section>
   </main>
+  <footer class="">
+    <h2 id="contact">Contact me</h2>
+    <p>
+      If you have an application you are interested in developing, a feature that you need built or a project that needs coding.
+      <br>
+      I’d love to help with it!
+    </p>
+    <form id="contact-form" action="https://formspree.io/f/xgedkalg" method="POST">
+      <input type="text" name="name" placeholder="Lorem Ipsum" maxlength="30" required>
+      <input type="email" name="email" id="email" placeholder="loremipsum@gmail.com" required>
+      <textarea name="message" id="message" placeholder="Write your message here..." cols="30" rows="10" maxlength="500" required></textarea>
+      <button class="btn" form="contact-form" type="submit">Get in touch</button>
+    </form>
+    <hr>
+  </footer>
```

Comment:
- [ ] 👍 It's important to make your contact form section responsive. Currently, the webpage seems to have a fixed size suitable for a 375px device width, but it's essential to make it responsive across different screen sizes. Please ensure that the form adapts and looks good from 375px to a minimum desktop screen size of 768px.

[Rio’s-Portfolio.webm](https://github.com/razvanso/rio/assets/55840999/feb1559c-5297-4a96-ae47-bac5d1c2242a)

- [ ] 👍 All elements in the contact form section currently have fixed width and height. Instead of using fixed values, it's recommended to use responsive values in your CSS rules, such as percentages (e.g., width: 80%), to ensure that the layout adjusts properly on different devices and screen sizes.

Reply:
**Thanks for pointing this out**, @shahadat3669 - I set the fixed dimension to more easily match the design, then forget about them, and of course everything seemed alright... :angel: :+1: Lucky we have vigilant code reviewers :pray: